.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 'Feature request'
+about: Suggest an idea for improving Splink
+title: '[FEAT] <title>'
+labels: 'enhancement'
+assignees: ''
+---
```

Comment:
These fields probably need updating

Reply:
Good spot. This is why templates are good - to stop idiots like me messing things up!